<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sensorWidget</class>
 <widget class="QWidget" name="sensorWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sensorWidget</string>
  </property>
  <layout class="QGridLayout" name="sensorLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="cameraSelectFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="cameraSelectLayout">
      <item>
       <widget class="QLabel" name="cameraLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Camera:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="camCombo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxVisibleItems">
         <number>12</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>26</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>CAM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAM12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ALLCAM</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="triggerBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Trigger</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="gainExposureFrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QSpinBox" name="gainSpinBox">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="gainLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Gain:</string>
          </property>
          <property name="buddy">
           <cstring>gainSlider</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="exposureLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Exp:</string>
          </property>
          <property name="buddy">
           <cstring>exposureSlider</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="exposureSlider">
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="exposureSpinBox">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="gainSlider">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="getValueBtn">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>GetVaule</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="laserBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>LaserButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="displayBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>DisplayImage</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="videoCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>VideoMode</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>gainSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
